src/Directory.Build.props

```diff
@@ -0,0 +1,9 @@
+<Project>
+  <PropertyGroup>
+    <DebugType>portable</DebugType>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="SourceLink.Create.CommandLine" Version="2.1.0" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.NuGet.Build.Tasks.Workaround" Version="*" PrivateAssets="all" />
```

Comment:
workaround for mixed msbuild projects like this for current MSBuild 15 https://github.com/NuGet/Home/issues/4532